### PR TITLE
update image resizer and postcoder

### PR DESCRIPTION
**Pull request by danieldevlewis**
- image-resizer
  - update app version
  - stop overriding the app version with "main"
  - include env mapping
  - include a secrets store
- postcoder
  - update app version
  - stop overriding the app version with "main"
  - remove env running as "development"

### CODE REVIEW

File: charts/postcoder-proxy/values.yaml [hunk 1]
```diff
@@ -14,15 +14,12 @@ externalSecrets: &external_secrets_config
 
 env:
   APP_ENV: development
-  NEW_RELIC_AGENT_ENABLED: 'false'
```
Comment (danieldevlewis) [inline charts/postcoder-proxy/values.yaml:17]: This isn't needed.  I've updated the app to only enable new relic if the key is present as an enc

File: charts/image-resizer/templates/deployment.yaml [hunk 2]
```diff
@@ -56,6 +56,13 @@ spec:
               scheme: HTTP
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          envFrom:
+          - configMapRef:
+              name: {{ include "image-resizer.fullname" . }}
```
Comment (michelesr) [inline charts/image-resizer/templates/deployment.yaml:61]: does this configmap exist? it doesn't look like it

File: charts/postcoder-proxy/values.yaml [hunk 3]
```diff
@@ -13,16 +13,12 @@ externalSecrets: &external_secrets_config
   stores: {}
 
 env:
-  APP_ENV: development
```
Comment (danieldevlewis) [inline charts/postcoder-proxy/values.yaml:16]: Not sure why we were doing this?

**Review by michelesr — CHANGES_REQUESTED** [review]

File: charts/image-resizer/templates/deployment.yaml (hunk 2 above)
Comment (danieldevlewis) [reply charts/image-resizer/templates/deployment.yaml:61]: Oh yes, I've added it now

File: charts/postcoder-proxy/values.yaml (hunk 3 above)
Comment (MatthewCane) [reply charts/postcoder-proxy/values.yaml:16]: I honestly don't remember, I think I was having issues with new relic running despite being disabled so the automated testing for the chart works properly without having to have a key. If the app has been updated to disable itself properly when a key is not present then this line can be removed.

File: charts/postcoder-proxy/values.yaml (hunk 1 above)
Comment (michelesr) [reply charts/postcoder-proxy/values.yaml:17]: if you remove all the keys in the yaml please put `{}` instead

Comment (danieldevlewis) [reply charts/postcoder-proxy/values.yaml:17]: Still a couple more in there

File: charts/postcoder-proxy/values.yaml (hunk 3 above)
Comment (danieldevlewis) [reply charts/postcoder-proxy/values.yaml:16]: Make sense.  It should be setup correctly now.

Comment (danieldevlewis) [reply charts/postcoder-proxy/values.yaml:16]: 🤞

**Review by michelesr — APPROVED** [review]

**Review by MatthewCane — APPROVED** [review]

**Pull request merged**